packages/ledgerstate/transaction.go

```diff
@@ -125,6 +125,18 @@ func (t TransactionIDs) Clone() (transactionIDs TransactionIDs) {
 	return
 }
 
+// Strings returns a slice of string rapresentation of the TransactionIDs.
+func (t TransactionIDs) Strings() (transactionIDs []string) {
+	transactionIDs = make([]string, len(t))
+	i := 0
+	for transactionID := range t {
+		transactionIDs[i] = transactionID.Base58()
+		i++
+	}
```

Comment:
You can initialize `transactionIDs` slice with predefined _capacity_ instead of _size_. This way you can `append` to that slice efficiently without reallocations and you don't need the `i` counter variable  
```suggestion
	transactionIDs = make([]string, 0, len(t))
	for transactionID := range t {
		transactionIDs = append(transactionIDs, transactionID.Base58())
	}
```